sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedField.cs

```diff
@@ -49,7 +49,7 @@ internal ExtractedField(KeyValuePair_internal field, ReadResult_internal readRes
 
         /// <summary>
         /// </summary>
-        public IReadOnlyList<RawExtractedItem> NameRawExtractedItems { get; internal set; }
+        public IReadOnlyList<FormContent> NameRawExtractedItems { get; internal set; }
```

Comment:
Would it make sense to update the property names here as well, or will that be addressed in the next phase?  (I'm fine either way)

Reply:
I intend to do it in a next phase. To keep track of what still needs to be updated, I'll rename a file only when it doesn't need more API changes.

Reply:
Sounds great.